mem-16-t.xml

```diff
@@ -3850,7 +3850,7 @@ This word takes the suffix {-Du':n:suff} when it refers to a body part, and {-me
       <column name="examples_zh_HK"></column>
       <column name="examples_pt"></column>
       <column name="search_tags">contradict, disavow, claim as false</column>
-      <column name="search_tags_de">widersprechen, ablehnen, als falsch behaupten</column>
```

Comment:
Wouldn't it still be useful to keep the search tags?

Reply:
Sure, why not.